TelCo.ColorCoder/GetColor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+    public static class GetColor
+    {
+        /// <summary>
+        /// Given a pair number function returns the major and minor colors in that order
```

Comment:
This comment has the same information as can be concluded by reading the function name and parameters.
The out-of-range exception has been left out of the comment, which means callers need to dig through the implementation to figure it out.

Reply:
okay...will update.